### PR TITLE
SAK-49860 Fix Custom Icon (FontAwesome Icon Picker Popup) Issues in Dark Mode and Add Placeholder Text

### PR DESCRIPTION
**Pull request by kunaljaykam**
https://sakaiproject.atlassian.net/browse/SAK-49860

image

### CODE REVIEW

File: library/src/skins/default/src/sass/tool.scss [hunk 1]
```diff
@@ -255,9 +255,23 @@ label {
...
-// basic-lti tool 
+// FontAwesome Icon Picker //
 .iconpicker-popover {
     opacity: 1 !important;
+    border-radius: 1px;
+}
+
+.iconpicker-popover .popover-title,
+.iconpicker-popover .iconpicker-items {
+    background-color: var(--sakai-background-color-2) !important;
```
Comment (adrianfish) [inline library/src/skins/default/src/sass/tool.scss:266]: The usual question: Do you really need this !important on here.

**Review by adrianfish — CHANGES_REQUESTED** [review]

File: library/src/skins/default/src/sass/tool.scss (hunk 1 above)
Comment (kunaljaykam) [reply library/src/skins/default/src/sass/tool.scss:266]: I should have mentioned it. We need it, or else Fontawesome would overwrite it.

Comment (adrianfish) [reply library/src/skins/default/src/sass/tool.scss:266]: Makes perfect sense. Thanks.

**Review by adrianfish — APPROVED** [review]
GTG. Thanks.

**Pull request merged**